mono/tests/coop-array.mak

```diff
@@ -0,0 +1,27 @@
+# This file is not used in automated tests, but is useful for unautomated testing.
+
```

Comment:
I'd like to see a comment here about making sure `MONO_DEBUG_ICALLARRAY` is defined in the source and `MONO_LOG_MASK=icallarray` is set.  Otherwise it's not obvious what is expected from running this manually.